src/oas3/__tests__/__fixtures__/id/legacy.ts

```diff
@@ -200,19 +200,19 @@ export default [
       path: [
         {
           // hash(`http_path_param-${parentId}-${param.name}`)
-          // This was defined on the path, so we use the path to generate the id (thus if another operation on this path was in this doc, it would have path param with same id)
+          // This was defined on the path, so we use the *operation* to generate the id (thus if another operation on this path was in this doc, it would have path param with a different id)
           // path's id = hash(`http_path-${parentId}-${path}`)
           // The closest parent id to a path, is the service, so this equals... (remember that path segments have characters removed, since they are basically meaningless)
```

Comment:
we may want to update this comment too

```suggestion
          // The closest parent id to a operation, is the service, so this equals... (remember that path segments have characters removed, since they are basically meaningless)
```